doc/Caliptra.md

```diff
@@ -664,113 +664,110 @@ The SoC may support a fuse bank for representing the hash of the owner's public
+| 001b                                           | DebugUnlocked and production    | This state is used when debugging of Caliptra RTM is required. When in this state: <ul><li>UDS and other identity critical assets are programmed into fuses. They may not have been written into Caliptra fuse registers if the unsecure state entered before Caliptra is out of reset. If the unsecure state transition happened after fuses are written to Caliptra, they are cleared when the security state transitions from secure/manufacturing -> unsecure.</li><li>Caliptra state: All security assets are in debug mode (UDS and obfuscation key are in production state).<ul><li>UDS: Reverts to a ‘well-known’ debug value.</li><li>Obfuscation key: Switched to debug key.</li><li>Key Vault is also cleared.</li><li>Caliptra JTAG is unlocked and allows microcontroller debug.</li><li>Caliptra JTAG can access IP internal registers through FW or directly.</li></ul></li></ul> | Debug unlocked -> debug locked is possible ONLY with a power cycle.            |
 
-**Note:** Other encodings are reserved and always assumed to be in a secure state.
+**Notes:** 
+* End of life state is owned by SoC. In end-of-life device lifecycle state, Caliptra shall not not be brought out of reset.
+* Other encodings are reserved and always assumed to be in a secure state.
 
-Each of these security states may be mapped to different SOC level debug and security states. SOC’s requirement is that if the SOC enters a debug state, then Caliptra must also be in Unsecured state where all assets are cleared.
+Each of these security states may be mapped to different SoC level debug and security states. SoC’s requirement is that if the SoC enters a debug state, then Caliptra must also be in an unsecured state where all assets are cleared.
 
-## Service Surface
+## Service surface
 
 The service surface of a Caliptra RTM has multiple vectors. All use cases are control plane services, useful to power on a system or start a task. Supporting line rate high performance IO cryptography or any other data path capability is not required.
 
-* **Logic IOs:** required to indicate status of the IP, availability of a message through APB, and to enable/disable certain debug capabilities (like JTAG enable/disable)
+* **Logic IOs:** Required to indicate status of the IP, availability of a message through APB, and to enable and disable certain debug capabilities (like JTAG enable or disable).
```

Comment:
I think "or" would be more appropriate here too

Reply:
fixed